Makefile

```diff
@@ -176,9 +176,11 @@ $(BINDIR)/openapi-gen: vendor/k8s.io/kubernetes/cmd/libs/go2idl/openapi-gen
 # Some prereq stuff
 ###################
 
-.init: $(scBuildImageTarget)
+.init: $(scBuildImageTarget) glide.lock
 	touch $@
 
+glide.lock: glide.yaml
```

Comment:
@pmorie I removed this target and added an explicit "runglide" step so that when we do need to run glide people can do via the Makefile instead of running it from the cmd line. This is good for those who don't have glide installed.  This new step is not added as a dependency on any target so its just there to be called manually by a dev.